### ISSUE TITLE
Make users aware that there resource location has been changed.

### ISSUE THREAD

**Issue by shresthagrawal** [ununu-p2p/ink-desktop#25]
It is important to make musicians aware that there is copy of there resource files is maintained and `project_dir/resource` and any modification made to previous resource files will not affect the project.

This problem in future could be smartly tackled by monitoring both the resource files and asking user on different scenarios. 

For now it would be enough to inform the user and also have a way to open the resource folder with one click from the project page.

**Comment by falafeljan** [comment]
good point. we could add a dialog that is posed before synchronizing changes, something like: "We need to copy all external files into the project directory. Keep existing files? [yes (copy)/no (move)]".

we could check if there are any external files at all before showing that dialog.
